abb_resources/package.xml

```diff
@@ -0,0 +1,22 @@
+<package>
+  <name>abb_resources</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      Shared configuration data for ABB manipulators.
+    </p>
+    <p>
+      This package contains common URDF / XACRO resources used by
+      ABB related packages.
```

Comment:
'style rules' (if we have those) say to keep *urdf* and *xacro* lower case (I know the `fanuc_resources` has this upper-cased, that needs to be fixed).